guidelines/1.3.1.md

```diff
@@ -1,57 +1,151 @@
+# Structure and relationships
 
-When content structures like tables, lists and headings are communicated visually, they must also be communicated in ways that assistive technologies can understand. This ensures that content structures are available to screen reader, screen magnifier and speech recognition tool users.
+> When designs convey a structure of information visually (like sections within a page, or a data table), that structure must also be identified in code so that screen readers can understand it.
 
-##### Requirements / What to do?
+## What?
 
-*   Tabular data is presented using proper HTML markup (`<table>`, `<tr>`, `<th>`, and `<td>` elements);
-*   Lists of items are presented using proper HTML markup (`<ul>`, `<ol>`, and `<li>` elements);
-*   Headings are presented using proper HTML markup (`<h1>` through `<h6>` elements);
-*   Form fields have properly associated `<label>` elements;
-*   Sets of radio buttons or checkboxes have a `<fieldset>` element to group them together, and a `<legend>` element to provide a label for the group;
-*   Form fields with format requirements or other additional information have properly associated hints.
+When content structures like tables, lists and headings are communicated visually, they must also be communicated in ways that assistive technologies can understand.
 
-##### Common mistakes
+### Requirements
 
-*   Tabular data is displayed using CSS to present the appearance of a table, but the proper HTML markup has not been used;
-*   A list of items is presented using line breaks to separate each item, and the proper HTML markup has not been used;
-*   Headings are text styled with CSS, and the proper HTML markup has not been used;
-*   Headings use the proper HTML markup, but do not accurately represent the hierarchy of content on the page (starting with `<h1>`, then `<h2>` for each section, `<h3>` for each sub-section and so on);
-*   Form fields do not have labels;
-*   Form fields have labels but they are not properly associated with their corresponding fields;
-*   Sets of radio buttons or checkboxes are not grouped inside a `<fieldset>` element, and do not have a `<legend>` element to provide a label for the group;
-*   Form fields have format requirements or additional information, but it is not properly associated with the corresponding field.
+* Headings are presented using the correct markup or traits;
+* Form fields have properly associated labels;
+* Form fields with format requirements or other additional information have properly associated hints;
+* Tabular data is presented using the correct markup (web only);
+* Lists of items are presented using the correct markup (web only);
+* Sets of radio buttons or checkboxes have a `<fieldset>` element to group them together, and a `<legend>` element to provide a label for the group (web only).
 
-##### Code examples
+### WCAG Success Criteria
```

Comment:
I'd like to avoid the acronym if possible, unless we define it further up on the page, or right below the header.

Not sure whether that's possible. Would be good to try.

E.g.
* the header could read "Official guideline"
* the content underneath could read: 'See [ 1.3.1 "Info and Relationships" in the Web Content Accessibility Guidelines](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)'

Or something like that.

Reply:
Ah yes I like that, I'll update existing places that use this

Reply:
Okay, updated, thoughts?